rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -27,6 +28,46 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 
 When a recording state changes, a toast notification appears indicating current state. The notification stays for two seconds and disappears automatically.
 
+== Image-Based Mouse Clicks
+
+Use image-based mouse clicks at any point during a normal recording session to record mouse clicks on a UI element that otherwise cannot be recorded properly. This feature guides you through the mouse click process. An image on the screen and mouse clicks must be captured, and the mouse clicks are recorded relative to the captured image.
+
+The feature also checks your display settings automatically while recording to ensure the feature records correctly.
+
+This feature will result in _Search Pattern_ and _Mouse Action_ Action Steps when the workflow is generated in RPA Builder from the recording.
+
+When using this feature:
+
+* The Recording Toolbar is inactive.
+* Keyboard events are not recorded.
+
+=== Recommended Display Settings
+
+Apply the following recommended display settings to capture image-based mouse clicks:
+
+* Single monitor. 
++
+Although the image-based mouse clicks feature supports multiple monitors, that setting limits the capturing to the primary display for Bot tasks and the web page for Web tasks.
+* Display scaling of 100%. If the scaling is different, then there is a risk that the captured image will not be found by the workflow generated in RPA Builder from the recording.
+
+To check your display settings before recording, go to *Windows Settings > Display*. If your *Scale and layout* or *Multiple displays* settings are different than what's recommended when recording, a warning message shows each time you record an image-based mouse click, and you must confirm the message.
```

Comment:
issue: if user wants to get rid of that warning, it is not enough to change the Windows Setting for Display, but also to restart the RPA Recorder completely (i.e. close the application and open it again). Please add a sentence saying that when changing the Windows Settings, user will need to restart the RPA Recorder for the changed Display settings to have any effect on the recording.